libs/SalesforceAnalytics/SalesforceAnalytics.xcodeproj/project.pbxproj

```diff
@@ -920,6 +922,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = CE2B8CAA1CE9281F00C6FC6A /* SalesforceAnalytics-Dynamic-iOS-Debug.xcconfig */;
 			buildSettings = {
+				APPLICATION_EXTENSION_API_ONLY = NO;
```

Comment:
After a close look at it the entry is related to SalesforceAnalyticsTestApp. Shouldn't be marked as YES.